Packs/ServiceNowITAdmin/Integrations/ServiceNow_IT_Admin/ServiceNow_IT_Admin.py

```diff
@@ -0,0 +1,515 @@
+                                               "errorCode", "errorMessage", "details"],
+                                      removeNull=True
+                                      )
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def enable_user_command(client, args):
+    """
+        Enable user using PATCH to Servicenow API , if Connection to the service is successful.
+
+        Args:   demisto command line argument
+        client: Service Client
+
+        Returns:
+            success : success=True, id, email, login as username, details, active status
+            fail : success=False, id, login as username, errorCod, errorMessage, details
+    """
+    scim = verify_and_load_scim_data(args.get('scim'))
+    parsed_scim_data = map_scim(scim)
+    user_id = parsed_scim_data.get('id')
+
+    if not (user_id):
+        raise Exception('You must provide sys id of the user')
+
+    custom_mapping = demisto.params().get('customMappingUpdateUser')
+    servicenow_user = client.build_servicenow_user_profile(args, scim, custom_mapping)
+    servicenow_user['active'] = True
+    servicenow_user['locked_out'] = False
+    res = client.update_user(user_id, servicenow_user)
+    res_json = res.json()
+
+    if res.status_code == 200:
+        result = res_json['result']
+        active = True if result['active'] == 'true' else False
+        id = result['sys_id']
+        email = result['email']
+        username = result['user_name']
+
+        generic_iam_context = OutputContext(success=True, iden=id, email=email,
+                                            username=username, details=result, active=active)
+    else:
+        generic_iam_context = OutputContext(success=False, iden=user_id, errorCode=res.status_code,
+                                            errorMessage=res_json.get('error', {}).get('message'), details=res_json)
+
+    generic_iam_context_dt = f'{generic_iam_context.command}(val.id == obj.id && val.instanceName == obj.instanceName)'
+    outputs = {
+        generic_iam_context_dt: generic_iam_context.data
+    }
+    readable_output = tableToMarkdown('Enable ServiceNow User:', t=generic_iam_context.data,
+                                      headers=["brand", "instanceName", "success", "active", "id", "username", "email",
+                                               "errorCode", "errorMessage", "details"],
+                                      removeNull=True
+                                      )
+    return (
+        readable_output,
+        outputs,
+        generic_iam_context.data
+    )
+
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    params = demisto.params()
+    api_version = params.get('api_version', None)
+
+    # get the service API url
+    base_url = urljoin(params.get('url').strip('/'), BASE_URI)
+    if api_version:
+        base_url += api_version
```

Comment:
why do we let the client choose the api version and not define it ourselves?

Reply:
@dantavori It's an optional parameter, if the user wants to run the particular version of API they needs to mention it here else latest will be called.